db/migrate/20160328221058_add_time_format_to_user.rb

```diff
@@ -0,0 +1,8 @@
+class AddTimeFormatToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :time_format, :string, :default => 'relative', :null => false
+    User.all.each do |user|
```

Comment:
also FYI `all.each` will blow up if there are too many records, so  `find_each` is usually better